python/triton/compiler/code_generator.py

```diff
@@ -1264,9 +1264,9 @@ def kernel_suffix(signature, specialization):
     suffix = ''
     for i, _ in enumerate(signature):
         suffix += str(i)
-        if i in specialization.equal_to_1:
+        if i in specialization["tt.equal_to_1"]:
             suffix += 'c'
-        if i in specialization.divisible_by_16:
+        if i in specialization["tt.divisibility"]:
```

Comment:
Do we need to have ways to handle additional properties injected by backends here?

Reply:
So I was a bit unsure about this. This method (apparently) is only used by the AOT driver. I can bake this in `AttrsProperty`, but I wanted to know if that was the right call. Because it looks that `AttrsProperty` is more dedicated for non-AOT/standard  compilation flow. 

Alternatively, we could move this method in `compile.py` and let different backends add their own AOT specialization in there.

Reply:
Yeah okay fine to leave it as-is for now.